resolwe_bio/tools/icount_demultiplex_annotate.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+"""Run iCount demultiplexing and annotate samples."""
+import argparse
+import gzip
+import resdk
+import re
+
+from time import sleep
+
+
+GENOME_MAP = {
+    'Hs': 'star-index-ens-hs',
+    'Mm': 'star-index-ens-mm'
+}
+
+SEGMENTATION_MAP = {
+    'Hs': 'icount-segmentation-hs',
+    'Mm': 'icount-segmentation-mm'
+}
+
+SPECIES_MAP = {
+    'Hs': 'Homo sapiens',
+    'Mm': 'Mus musculus'
+}
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Run iCount demultiplexing.")
+    parser.add_argument('annotation_file', help="iCount sample annotation file.")
+    parser.add_argument('annotation_id', type=int, help="Annotation file object ID")
+    parser.add_argument('reads_id', type=int, help="Reads file object ID")
+    parser.add_argument('data_id', type=int, help="Current data ID")
+    return parser.parse_args()
+
+
+def _tsv_to_dict(table_file):
```

Comment:
The same function in two files. Can you import the function from another module?